inc/post-formats.php

```diff
@@ -57,8 +57,8 @@ function eighties_post_format_video_first_video() {
  *      the_excerpt for these loops.
  *
  * @since 1.0.0
- * @var content
-*/
+ * @param string $content For the function.
```

Comment:
@n8finch "For the function." should change to something like "The post type's content." or just "The content."